# Провалює тест
def test_failure():
    assert True, "Тест завершується без помилки"
